### ISSUE TITLE
ESP8266 RTOS SDK

### ISSUE THREAD

**Issue by ivankravets** [platformio/platform-espressif8266#16]
# Moved from https://github.com/platformio/platformio/issues/402

/cc @andriyadi
-------

I'm about to adopt ESP8266 native SDK, and since my use cases require parallelism, it would be great to adopt RTOS SDK. I've found this project, [ESP-OPEN-RTOS](https://github.com/SuperHouse/esp-open-rtos), somehow better than the official Espressif's.

Would it be possible to support this kind of SDK into Platformio?

Thanks in advance. Have a great holiday guys!

**Comment by adikabintang** [comment]
Hi, is there any news about this? Actually I have tried [1] but I failed. After several hours investigating the problems, I think the latest version of ESP RTOS SDK [2] no longer fits with the provided python script in that branch. Currently, I am modifying the sming.py in order to work properly but unfortunately it still does not work. Has anybody recently tried SmingRTOS on PlatformIO and succeed?  

[1] https://github.com/platformio/platform-espressif8266/tree/feature/framework-sming 
[2] https://github.com/espressif/ESP8266_RTOS_SDK
